### ISSUE TITLE
Misplaced slide

### ISSUE THREAD

**Issue by tdutrion** [kevinjhappy/presentation-php-dates-revealjs#1]
I believe the slide [5.9](https://kevinjhappy.github.io/presentation-php-dates-revealjs/#/5/9) is not well positioned.

Please have a look and get back to me here :)

**Comment by kevinjhappy** [comment]
Yes after reviewing all slides in general, this one appears to be off topic, I need to put the error "last Month" behavior on the 31th of october in the dateTime traps, and then add my slide about carbon that correct this behavoir
Or I could make one seperate chapter to just talk about Carbon, which is a DateTime library and indicate this information.
Or not talk about carbon at all and stay with PHP native functions :)
